### DISCUSSION TITLE
can't download minios because the website does not allow me to see the whole page and there is no page navigation bar

### DISCUSSION THREAD

**Discussion by hdaz000700** [minios-linux/minios-live#10] [Q&A] [answered]
working on a laptop with a screen res of 1600 x 900

This is all I see attached evidance 
![minios_01](https://github.com/minios-linux/minios-live/assets/19715735/83d9da02-5d3e-4a22-81ec-22136c2b4e8c)

full screen
![minios_02](https://github.com/minios-linux/minios-live/assets/19715735/1d45ff16-2b74-4e7e-a9f5-22391fe03531)

would like to try but your website is not making it easy :P

**Comment by crim50n** [comment, marked as answer]
Just scroll below with your mouse wheel.
